salt/_modules/metalk8s_kubernetes_utils.py

```diff
@@ -159,7 +159,7 @@ def get_service_endpoints(service, namespace, kubeconfig):
             kind="Endpoints",
             apiVersion="v1",
             namespace=namespace,
-            kubeconfig=kubeconfig,
+            kwargs=kwargs,
```

Comment:
No you need to give it like this (otherwise you will just have a single `kwargs` argument containing the kwargs dict

```suggestion
            **kwargs,
```

Reply:
Fixed.